Firestore User Document - Error installing extension. Be aware that a parameter misconfiguration might have caused some extension resources

> Error installing extension. Be aware that a parameter misconfiguration might have caused some [extension resources](https://firebase.google.com/docs/extensions/overview-use-extensions?hl=en&authuser=0#resources) to not successfully deploy.
> View details
> Use the details below to help resolve this error, then retry installing the extension. If you continue to get this error, we recommend uninstalling the extension to remove any extension resources that might be deployed.
> 
> ; RESOURCE_ERROR at /deployments/firebase-ext-firestore-user-document/resources/backfillExistingUsers: {"ResourceType":"gcp-types/cloudfunctions-v1:projects.locations.functions","ResourceErrorCode":"400","ResourceErrorMessage":"Build failed: npm ERR! Cannot read property 'firebase-admin' of undefined\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR! /www-data-home/.npm/_logs/2023-05-17T12_36_15_593Z-debug.log; Error ID: beaf8772"}

--- a/firestore-user-document/CHANGELOG.md
+++ b/firestore-user-document/CHANGELOG.md
@@ -1,3 +1,7 @@
+## Version 0.0.7
+
+- Fix bugs
+
 ## Version 0.0.6
 
 - Fix bugs

--- a/firestore-user-document/extension.yaml
+++ b/firestore-user-document/extension.yaml
@@ -1,5 +1,5 @@
 name: firestore-user-document
-version: 0.0.6
+version: 0.0.7
 specVersion: v1beta
 displayName: Firestore User Document
 

--- a/firestore-user-document/functions/src/config.ts
+++ b/firestore-user-document/functions/src/config.ts
@@ -8,10 +8,10 @@ interface IConfig {
 
 const config: IConfig = {
   usersCollectionPath: process.env.USERS_COLLECTION_PATH!,
-  fieldsToPopulate: process.env.FIELDS_TO_POPULATE!.split(','),
+  fieldsToPopulate: process.env.FIELDS_TO_POPULATE!.split(","),
   deleteDocumentOnUserDelete:
-    process.env.DELETE_DOCUMENT_ON_USER_DELETE === 'true',
-  backfillExistingUsers: process.env.BACKFILL_EXISTING_USERS === 'true',
+    process.env.DELETE_DOCUMENT_ON_USER_DELETE === "true",
+  backfillExistingUsers: process.env.BACKFILL_EXISTING_USERS === "true",
   location: process.env.FUNCTIONS_LOCATION!,
 };
 

--- a/firestore-user-document/functions/src/index.ts
+++ b/firestore-user-document/functions/src/index.ts
@@ -1,7 +1,7 @@
 import { initializeApp } from "firebase-admin/app";
-import { getFirestore } from 'firebase-admin/firestore'
 import { UserRecord, getAuth } from "firebase-admin/auth";
 import { getExtensions } from "firebase-admin/extensions";
+import { getFirestore } from "firebase-admin/firestore";
 import { getFunctions } from "firebase-admin/functions";
 import * as functions from "firebase-functions";
 
@@ -9,6 +9,7 @@ import config from "./config";
 
 initializeApp();
 
+const auth = getAuth();
 const db = getFirestore();
 
 const usersCollection = db.collection(config.usersCollectionPath);
@@ -74,7 +75,7 @@ export const backfillExistingUsers = functions.tasks
     }
 
     try {
-      const { users, pageToken } = await getAuth().listUsers(
+      const { users, pageToken } = await auth.listUsers(
         BATCH_SIZE,
         data.pageToken
       );

